queries.sql

```diff
@@ -9,3 +9,28 @@ SELECT  * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
 
+/* Database Transactions */
```

Comment:
- [ ] @Nejjari001 great work on your transaction however you are missing the transaction below, kindly add the screenshot for the same:
![image](https://user-images.githubusercontent.com/61408860/209126605-817ef199-7522-4da4-ab21-a2f8f6861881.png)